CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -1173,6 +1173,11 @@ function Patient:afterLoad(old, new)
     self.under_infection_attempt = self.attempted_to_infect
     self.attempted_to_infect = nil
   end
+
+  if old < 213 and new >= 213 then
+    self.mood_marker = 2
```

Comment:
This should be `1` -- wondered why I was having such a headache yesterday 🤣

Reply:
new >= is unnecessary, new is the current version.

Reply:
Fixed

Reply:
NB: afterLoad style is missing from our coding conventions. I'll try to remember to add it next time I'm in the wiki.